dev/io.openliberty.microprofile.telemetry.1.0.internal/resources/io/openliberty/microprofile/telemetry/internal/resources/MPTelemetry.nlsprops

```diff
@@ -0,0 +1,30 @@
+#CMVCPATHNAME N/A
+#COMPONENTPREFIX CWMOT
+#COMPONENTNAMEFOR CWMOT MicroProfile Telemetry Tracing
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#ISMESSAGEFILE true
+# -------------------------------------------------------------------------------------------------
+#*******************************************************************************
+# Copyright (c) 2023 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+# This file follows the WebSphere Message Guidelines.
+# For more information, visit:
+# http://washome.austin.ibm.com/xwiki/bin/view/MessagesTeam/
+
+#-----------------------------------------------------------------------------------------------------------------------------
+# Message prefix block: CWMOT6000 - CWMOT5699
+#-----------------------------------------------------------------------------------------------------------------------------
+
+CWMOT6000.tracing.is.disabled=CWMOT6000W: MicroProfile Telemetry Tracing is disabled.
+CWMOT6000.tracing.is.disabled.explanation= By default, MicroProfile Telemetry Tracing is disabled. otel.sdk.disabled is not set to false.
+CWMOT6000.tracing.is.disabled.useraction= Specify otel.sdk.disabled=false in any of the configuration sources available via MicroProfile Config.
```

Comment:
We need a little more detail here about why tracing is disabled and what that means for the user.

```suggestion
CWMOT6000.tracing.is.disabled=CWMOT6000I: The MicroProfile Telemetry Tracing feature is enabled but has not been configured to create traces for the {0} application.
CWMOT6000.tracing.is.disabled.explanation=The MicroProfile Telemetry Tracing feature needs configuration to create traces and export them to a tracing system.
CWMOT6000.tracing.is.disabled.useraction=To enabled tracing, specify otel.sdk.disabled=false in any of the configuration sources available via MicroProfile Config. Further configuration may be required to allow MicroProfile Telemetry Tracing feature to export traces to your tracing system.
```